pkg/agent/datapath/policyBridge.go

```diff
@@ -357,13 +357,17 @@ func (p *PolicyBridge) RemoveLocalEndpoint(endpoint *Endpoint) error {
 }
 
 func (p *PolicyBridge) GetTierTable(direction uint8, tier uint8) (*ofctrl.Table, *ofctrl.Table, error) {
+	// POLICY_TIER0 for endpoint isolation policy:
+	// 1) high priority rule is whitelist for support forensic policyrule, thus packet that match
+	//    that rules should passthrough other policy tier ---- send to ctCommitTable;
+	// 2) low priority rule is blacklist for support general isolation policyrule.
```

Comment:
If in such case: VM1 and VM2 are in the same host and in the same VDS, VM1 with forensic policy: "VM1 allow ingress with VM2, VM2->VM1", but VM2 with isolation policyRule, what's the expect behavior?